src/helpers/__tests__/overrides.test.js

```diff
@@ -12,7 +12,7 @@ import {
   toObjectOverride,
   mergeOverrides,
   mergeOverride,
-  mergeStyleOverrides,
+  mergeSpecialOverrides,
```

Comment:
can we rename the utility to something like 'mergeConfigurationOverrides'? the fields props and style are not particularly special (they are two-thirds of the available fields). the difference i see between these fields and 'component' is that these provide small configurations while the other fully replaces